src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
One of the overflow checks could be removed, but this is not immediatley to see.
The condition `Columns - pum_col - pum_scrollbar >= INT_MIN` should be always true at this point. The `if` on line 215 tests for something similar and we know that `pum_scrollbar` is `0` or `1` which is never bigger than `PUM_DEF_WIDTH` or `max_width`.